src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_29">
+      <div class="calendar__day start-day_sun"></div>
```

Comment:
start-day should be a modifier for the calendar (like month-length)

```suggestion
    <div class="calendar calendar--month-length_29 calendar--start-day_sun">
      <div class="calendar__day"></div>
```